crates/bevy_ecs/src/schedule_v3/executor/multi_threaded.rs

```diff
@@ -183,9 +183,9 @@ impl SystemExecutor for MultiThreadedExecutor {
                 let world = unsafe { &mut *world.get() };
                 apply_system_buffers(&mut self.unapplied_systems, systems, world);
 
-                debug_assert!(self.ready_systems.is_clear());
-                debug_assert!(self.running_systems.is_clear());
-                debug_assert!(self.unapplied_systems.is_clear());
+                debug_assert!(self.ready_systems.is_empty());
+                debug_assert!(self.running_systems.is_empty());
+                debug_assert!(self.unapplied_systems.is_empty());
```

Comment:
Somehow the code from bevy main did not compile. This fixed it. What is going on?

Reply:
fixed